operations/validate_csm_health.md

```diff
@@ -397,43 +428,68 @@ If one or more checks failed:
 
 Additional test execution details can be found in `/opt/cray/tests/cmsdev.log`.
 
+## 4. NCN Gateway Health Checks
+
+The gateway tests check the health of the API Gateway on all of the relevant networks.   On NCNs, the API gateway is accessible 
+on the NMNLB network, the CMN network, and either the CAN or CHN user network depending on the configuration of the system.  
+The gateway tests will check that the gateway is accessible on all networks it should be accessible and NOT accessible on all 
+networks it should NOT be accessible.  It will also check several service endpoints to verify that they return the proper response 
```

Comment:
```suggestion
networks it should NOT be accessible. It will also check several service endpoints to verify that they return the proper response 
```